src/chrome/content/rules/Touchgen.net.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Touchgen" platform="mixedcontent">
```

Comment:
Yes, all the Javascript seems to load fully on HTTPS. The mixed content is on "https://mail.touchgen.net". Only a few images show the alt text. I added some more test URLs for the images that do not show up.